Content.Client/UserInterface/Systems/Actions/ActionUIController.cs

```diff
@@ -662,6 +660,11 @@ private void OnActionPressed(GUIBoundKeyEventArgs args, ActionButton button)
         if (args.Function != EngineKeyFunctions.UIClick)
             return;
 
+        HandleActionPressed(args, button);
+    }
```

Comment:
```suggestion
// Moffstation Start
        HandleActionPressed(args, button);
    }
// Moffstation End
```

Multiline changes get start/end comments.
But tbh this is kind of weird since the change cuts across semantic units, so I am not sure what the best way to comment this would be.

Reply:
just do the one line comment?

Reply:
Aye aye captain.

Reply:
Roomba previously was picky about even non-semantic lines getting comments. I'm good with just a comment on the function invocation line.

Reply:
My illiterate ass isn't sure what semantic means in this context

Reply:
A function invocation is "semantic" in that it "means something".
Just a brace is "syntactic" in that it's arcane sigils needed to make the code work, but doesn't inherently _mean_ anything.

Any individual set of tokens in legal code can be sorted into one or the other category.
(don't bug me about `(` and `)` in a function binding being purely syntactic when the overall binding is semantic)